web/client/components/geostory/contents/ToolbarButtons.jsx

```diff
@@ -106,3 +109,16 @@ export const DeleteButtonToolbar = ({ editMap: disabled = false, path, remove =
             remove(path);
         }} />
     );
+    // { align, sectionType, size, src, update = () => { } }
+export const EditURLButtonToolbar = (props) => {
+    return (
+        <ToolbarDropdownInputButton
+            glyph="pencil"
+            pullRight={(props.align === "right" || props.size === "full" || props.size === "large") && !props.sectionType}
```

Comment:
@kboczkowski-clurgo i think this  is not needed and since it is a toggle button you can copy the editMedia button
this was created for buttons used as dropdowns with a menu list
please remove this and put it in the content toolbar as editMedia

btw this was missing a test